service/src/main/resources/db/changelog/changesets/20240228_add_fence_account_key_and_dist_lock_table.yaml

```diff
@@ -0,0 +1,52 @@
+databaseChangeLog:
+  - changeSet:
+      id: "add_fence_account_key_and_dist_lock_table"
+      author: sehsan
+      changes:
+        - createTable:
+            tableName: fence_account_key
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: linked_account_id
```

Comment:
Oh! Can we make this cascade on delete? That way, if someone deletes their linked account, the key will automatically get deleted too.

Reply:
Oh I assumed that was the default behavior 😅 I'll double check!